code/script.js

```diff
@@ -225,9 +228,10 @@ const CHARACTERS = [
 ]
 
 // Global variables
-let secret, currentQuestion, charactersInPlay
+let secret, currentQuestion, charactersInPlay, secretValue
+let countClicks = 0;
```

Comment:
you initialized it here as 0 and then set it again to 0 in the `start()`. You could create this variable with others one line up 😸